Riot/Modules/Room/RoomInfo/RoomInfoList/RoomInfoListViewModel.swift

```diff
@@ -33,7 +33,9 @@ final class RoomInfoListViewModel: NSObject, RoomInfoListViewModelType {
     weak var coordinatorDelegate: RoomInfoListViewModelCoordinatorDelegate?
     
     private var viewData: RoomInfoListViewData {
-        let encryptionImage = EncryptionTrustLevelBadgeImageHelper.roomBadgeImage(for: room.summary.roomEncryptionTrustLevel())
+        let encryptionImage = room.summary.isEncrypted ?
```

Comment:
We can check if it worth to add a `RoomEncryptionTrustLevelNone` to `RoomEncryptionTrustLevel` enum. To be sure to have dedicated value when room is not encrypted like `UserEncryptionTrustLevel`.

Reply:
I also considered that before, it's probably the right thing to do if you also thought about it :). Updated.